sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/properties/AzureServiceBusJmsProperties.java

```diff
@@ -65,70 +65,138 @@
      */
     private String password;
 
+    /**
+     * Get the URL of the AMQP broker.
+     * @return the URL of the AMQP broker.
+     */
     public String getRemoteUrl() {
         return remoteUrl;
     }
 
+    /**
+     * Set the URL of the AMQP broker.
+     * @param remoteUrl the URL of the AMQP broker.
+     */
     public void setRemoteUrl(String remoteUrl) {
         this.remoteUrl = remoteUrl;
     }
 
+    /**
+     * Get the login user of the AMQP broker.
+     * @return the login user of the AMQP broker.
+     */
```

Comment:
why need comments in both field and getters at the same time?

Reply:
The architects ask us to do so.